lib/client.js

```diff
@@ -585,14 +653,65 @@ MqttClient.prototype.unsubscribe = function (topic, callback) {
  *
  * @api public
  */
-MqttClient.prototype.end = function (force, cb) {
+MqttClient.prototype.end = function () {
   var that = this
+  // var errors = {
+  //   0: '',
+  //   4: 'Disconnect with Will Message',
+  //   128: 'Unspecified error',
+  //   129: 'Malformed Packet',
+  //   130: 'Protocol Error',
+  //   131: 'Implementation specific error',
+  //   135: 'Not authorized',
+  //   137: 'Server busy',
+  //   139: 'Server shutting down',
+  //   141: 'Keep Alive timeout',
+  //   142: 'Session taken over',
+  //   143: 'Topic Filter invalid',
+  //   144: 'Topic Name invalid',
+  //   147: 'Receive Maximum exceeded',
+  //   148: 'Topic Alias invalid',
+  //   149: 'Packet too large',
+  //   150: 'Message rate too high',
+  //   151: 'Quota exceeded',
+  //   152: 'Administrative action',
+  //   153: 'Payload format invalid',
+  //   154: 'Retain not supported',
+  //   155: 'QoS not supported',
+  //   156: 'Use another server',
+  //   157: 'Server moved',
+  //   158: 'Shared Subscriptions not supported',
+  //   159: 'Connection rate exceeded',
+  //   160: 'Maximum connect time',
+  //   161: 'Subscription Identifiers not supported',
+  //   162: 'Wildcard Subscriptions not supported'
+  // }
```

Comment:
why is this block commented here? If it's not needed, remove.

Reply:
resolve